pkg/util/iptables/iptables.go

```diff
@@ -575,3 +576,252 @@ func IsNotFoundError(err error) bool {
 	}
 	return false
 }
+
+type RuleOption struct {
+	Arg     string
+	Negated bool
+}
+
+type Rule struct {
+	Modules []string
+	Jump    Chain
+	Comment string
+	Options map[string]RuleOption
+}
+
+// Split a line into space-separated arguments, handling simple quoting
+func splitLine(line string) ([]string, error) {
+	items := strings.Split(line, " ")
+	args := make([]string, 0, len(items))
+	for i := 0; i < len(items); i++ {
+		if items[i] == "" {
```

Comment:
@thockin strings.Split("    ", " ") = [ '', '', '', '' ].  Not particularly useful, but that's a pretty common string split pattern.